packages/appkit/src/tests/appkit.test.ts

```diff
@@ -322,18 +321,18 @@ describe('Base', () => {
     })
 
     it('should get approved CAIP network IDs', () => {
-      vi.mocked(NetworkController.getApprovedCaipNetworkIds).mockReturnValue(['eip155:1'])
+      vi.mocked(ChainController.getAllApprovedCaipNetworks).mockReturnValue(['eip155:1'])
```

Comment:
Is it renamed to `getAllApprovedCaipNetworks` instead of `getAllApprovedCaipNetworkIds` (previously it was `*Ids`)

Reply:
renamed to be consistent